### DISCUSSION TITLE
NameError: name 'interpret_model' is not defined,  NameError: name 'predict_model' is not defined

### DISCUSSION THREAD

**Discussion by yts61** [pycaret/pycaret#1302] [General]
I am using python 3.9.0, and the full installation of pycaret 2.3. When i run interpret_model(model), i am still getting error :
    NameError: name 'interpret_model' is not defined

same problem with predict_model(model, data)
    NameError: name 'predict_model' is not defined

any solutions for these?

**Comment by Yard1** [comment]
Have you imported pycaret functions?

**Reply by Yard1** [reply]
Can you post the entire code?
